docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -459,6 +459,9 @@ The connector supports pushdown for a number of operations:
 Predicate pushdown support
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Predicates are pushed down for most types, including ``UUID`` and temporal
+types, such as ``DATE`` and ``TIME``.
```

Comment:
This reflects what @findepi mentioned in the slack communication and is good imho. Ideally we have a real list of supported data type or a statement like "for all types, besides xyz."

Reply:
TIME pushdown is not supported.
 @sheajamba @mosabua 
you should be able to validate soundness of docs with EXPLAIN over example tables.
At this point, the docs should already cover all the necessary steps to do so.

Reply:
Explicitly documenting which types are and are not supported is error prone and complex since pushdown depends on a lot of other factors as well.

We should instead teach users how to identify whether something was or wasn't pushed down by reading the EXPLAIN plan. It scales much better.

Reply:
Sure .. we have that to some degree in https://trino.io/docs/current/optimizer/pushdown.html and we should add more in https://trino.io/docs/current/optimizer/pushdown.html#predicate-pushdown

But .. we also need to add info that helps once a user finds that something is not pushed down .. then they have to be able to understand why .. is there some config problem, do have to do that.. or is it just not supported..